internal/docker/cypress.go

```diff
@@ -17,7 +20,7 @@ type CypressRunner struct {
 }
 
 // NewCypress creates a new CypressRunner instance.
-func NewCypress(c cypress.Project, ms framework.MetadataService, wr job.Writer, jr job.Reader, dl download.ArtifactDownloader, reps []report.Reporter) (*CypressRunner, error) {
+func NewCypress(c cypress.Project, regio region.Region, slSvc slack.Service, ms framework.MetadataService, wr job.Writer, jr job.Reader, dl download.ArtifactDownloader, reps []report.Reporter) (*CypressRunner, error) {
```

Comment:
I'm not sure to get where there is a `regio region.Region` field.
Is it there for future usage ?

Reply:
I was using `regio region.Region` for building URL, but URL now comes in a proper format and I forgot to delete region variable. I fixed this issue.